internal/cephfs/groupcontrollerserver.go

```diff
@@ -735,21 +735,11 @@ func (cs *ControllerServer) DeleteVolumeGroupSnapshot(ctx context.Context,
 	}
 	vgo.Destroy()
 
-	volIds := vgsi.GetVolumeIDs()
-	fsMap, err := getFsNamesAndSubVolumeFromVolumeIDs(ctx, req.GetSecrets(), volIds, cr)
-	err = extractDeleteVolumeGroupError(err)
-	if err != nil {
-		log.ErrorLog(ctx, "failed to get volume group options: %v", err)
-		err = extractDeleteVolumeGroupError(err)
-		if err != nil {
-			return nil, status.Error(codes.Internal, err.Error())
-		}
-
-		return &csi.DeleteVolumeGroupSnapshotResponse{}, nil
-	}
-
-	defer destroyFSConnections(fsMap)
-
+	// deleteSnapshotsAndUndoReservation will reset the quiesce of the subvolumes.
+	// In case of DeleteVolumeGoupSnapshot, reset of quiesce will extend the quiesce
+	// timer and result in IO freeze until the timer expires.
+	// Passing empty fsMap, deleteSnapshotsAndUndoReservation will skip resetting the quiesce.
+	fsMap := map[string]core.FSQuiesceClient{}
```

Comment:
Please define this as a const (just above `deleteSnapshotsAndUndoReservation()`) and document the behavior at the function.

Having a dedicated type for `map[string]core.FSQuiesceClient{}` would improve readability even more. Not sure how easy that would be, it's a nice bonus if you can do that.